base-image/assets/prepare_version.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#see man login(1) and ENVIRON(7)
+SHELL=/bin/bash
+USER=travis
+LOGNAME=travis
+LC_MESSAGES=POSIX
+LANG=en_US.UTF-8
```

Comment:
not sure...
Make sence to move settings this variables to env.sh and here simply `source env.sh` ?

prefer to use "default value", e.g. `USER=${USER:-"travis"}`